components/Footer.js

```diff
@@ -1,45 +1,45 @@
-import Image from "next/image";
 import Link from "next/link";
 import styled from "styled-components";
+import { BookmarkIcon } from "./BookmarkButton";
+import ScheduleIcon from "./Icons";
 
 export default function Footer() {
   return (
     <StyledFooterList>
       <StyledIconContainer>
         <Link href={"/"}>
-          <Image
+          <ScheduleIcon
             aria-label="overview"
             alt="overview"
-            src={"/assets/calendar.svg"}
-            width={20}
-            height={20}
+            fill="var(--biber-blue-color)"
+            width={40}
+            height={40}
           />
         </Link>
       </StyledIconContainer>
       <StyledIconContainer>
         <Link href={"/bookmarks"}>
-          <Image
+          <BookmarkIcon
             aria-label="bookmark"
             alt="bookmark"
-            src={"/assets/bookmark-filled.svg"}
-            width={20}
-            height={20}
+            color="#245669"
+            width={40}
+            height={40}
           />
         </Link>
       </StyledIconContainer>
     </StyledFooterList>
   );
 }
 
-const StyledFooterList = styled.ul`
+const StyledFooterList = styled.footer`
```

Comment:
I'm not sure if "StyledFooterList" is misleading because it's a footer and not actually a list. Maybe you could rename it to only "StyledFooter"?

Reply:
True. Will change with next commit.